docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,544 @@
+
+When the command is successful, you will see logs of `configtxgen` loading the `configtx.yaml` file and printing a channel creation transaction:
+```
+INFO 001 Loading configuration
+INFO 002 orderer type: etcdraft
+INFO 003 Loaded configuration: /fabric/config/configtx.yaml
+INFO 004 Generating genesis block
+INFO 005 Writing genesis block
+
+```
+
+## Step two: Use the `osnadmin` CLI to start the channel
+
+Now that the genesis block has been created, the first ordering node that runs the `osnadmin channel join` command effectively "activates" the channel, though the channel is not fully operational until a quorum of consenters is established (if your profile listed three consenters, at least one more node, for a total of two, would have to join using the `osnadmin channel join` command).
+
+**Note:** If you try to run the `osnadmin` commands (aside from the `channel list` command) against an ordering node that is a member of a system channel, you get an error which indicates that the system channel still exists. The system channel needs to be [removed](#remove-the-system-channel) before the `osnadmin` commands can be used to create or join channels.
+
+Each orderer needs to run the following command:
+
+```
+export TLS_CA_ROOT_CERT=../config/organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls/cacerts/tls-ca-cert.pem
+export OSN_TLS_SIGN_CERT=../config/organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls/signcerts/cert.pem
+export OSN_TLS_PRIVATE_KEY=../config/organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls/keystore/key.pem
+
+osnadmin channel join –-channel-id [CHANNEL_NAME]  --config-block [CHANNEL_CONFIG_BLOCK] -o [ORDERER_ADMIN_LISTENADDRESS] --ca-file $TLS_CA_ROOT_CERT --client-cert $OSN_TLS_SIGN_CERT --client-key $OSN_TLS_PRIVATE_KEY
+```
+
+Replace:
+- `CHANNEL_NAME` with the name you want to call this channel.
+- `CHANNEL_CONFIG_BLOCK` with the path and file name of the genesis block you created in Step one if you are creating the channel. Each subsequent ordering node can join the configuration starting with the genesis block, or they can join by providing the latest config block instead.
+- `ORDERER_ADMIN_LISTENADDRESS` corresponds to the `Orderer.Admin.ListenAddress` defined in the `orderer.yaml` for this orderer.
+- `TLS_CA_ROOT_CERT` with the path and file name of the orderer organization TLS CA root certificate and intermediate certificate if using an intermediate TLS CA.
+- `OSN_TLS_SIGN_CERT` with the path and file name of the orderer signed certificate from the TLS CA.
+- `OSN_TLS_PRIVATE_KEY` with the path and file name of the orderer private key from the TLS CA.
```

Comment:
This should be the admin client's certificate and private key, not the server's.